.github/workflows/go.yaml

```diff
@@ -0,0 +1,23 @@
+name: Go
+
+on: push
+
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: 1.15
```

Comment:
We have to make sure the code is still compile with gotip, see `ci/run` script for details.

Reply:
I try to use gotip by `gotip download` but it kinda sl